dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralIT.java

```diff
@@ -1162,8 +1197,31 @@ public void testGrabChecksumFromGitLab() {
         verifyChecksumsAreSaved(updatedTags);
     }
 
+    @Test
+    public void testGrabChecksumFromGitHubContainerRegistry() {
+        final ApiClient webClient = getWebClient(USER_2_USERNAME, testingPostgres);
+        ContainersApi toolApi = new ContainersApi(webClient);
+        ContainertagsApi toolTagsApi = new ContainertagsApi(webClient);
+        DockstoreTool tool = registerManualGitHubContainerRegistryToolAndAddTag();
+        List<Tag> tags = toolApi.getContainer(tool.getId(), null).getWorkflowVersions();
+        verifyChecksumsAreSaved(tags);
```

Comment:
not strictly in new code but `verifyChecksumsAreSaved` can pass if given an empty array, in at least case should check that it is not empty